docker/test/stateless/run.sh

```diff
@@ -57,6 +57,7 @@ if [[ -n "$BUGFIX_VALIDATE_CHECK" ]] && [[ "$BUGFIX_VALIDATE_CHECK" -eq 1 ]]; th
     sudo mv /etc/clickhouse-server/config.d/zookeeper.xml.tmp /etc/clickhouse-server/config.d/zookeeper.xml
 
     # it contains some new settings, but we can safely remove it
+    rm /etc/clickhouse-server/config.d/handlers.yaml
```

Comment:
It is placed here (not at the end) to avoid conflicts with https://github.com/ClickHouse/ClickHouse/pull/60375/files